app/src/main/cpp/native-lib.cpp

```diff
@@ -0,0 +1,67 @@
+#include <jni.h>
+#include <android/log.h>
+#include <wallet.h>
+#include <string>
+#include <math.h>
+
+#include "../../../../jniLibs/wallet.h"
+#include <android/log.h>
+
+#define LOG_TAG "Tari Wallet"
+
+/**
+ * Log functions. Log example:
+ *
+ * int count = 5;
+ * LOGE("Count is %d", count);
+ * char[] name = "asd";
+ * LOGI("Name is %s", name);
+ */
+#define LOGE(...) __android_log_print(ANDROID_LOG_ERROR,    LOG_TAG, __VA_ARGS__)
+#define LOGW(...) __android_log_print(ANDROID_LOG_WARN,     LOG_TAG, __VA_ARGS__)
+#define LOGI(...) __android_log_print(ANDROID_LOG_INFO,     LOG_TAG, __VA_ARGS__)
+#define LOGD(...) __android_log_print(ANDROID_LOG_DEBUG,    LOG_TAG, __VA_ARGS__)
+
+extern "C"
+JNIEXPORT jstring JNICALL
+Java_com_tari_android_wallet_ui_activity_MainActivity_stringFromJNI(
+        JNIEnv *env,
+        jobject /* this */) {
+    std::string hello = "Message from JNI!";
+    return env->NewStringUTF(hello.c_str());
+}
+
+extern "C"
+JNIEXPORT void JNICALL
+Java_com_tari_android_wallet_ui_activity_MainActivity_changeCountJNI(
```

Comment:
This method can be removed

Reply:
Just included this and stringFromJNI for demo purposes, deleting them.